src/services/findAllReferences.ts

```diff
@@ -1126,6 +1130,14 @@ export namespace Core {
                 // import("foo") with no qualifier will reference the `export =` of the module, which may be referenced anyway.
                 return nodeEntry(reference.literal);
             }
+            else if (reference.kind === "implicit") {
+                // Return either: The first JSX node in the file, the first statement of the file, or the whole file if neither of those exist
+                const range = forEachChildRecursively(
+                    reference.referencingFile,
+                    n => !(n.transformFlags & TransformFlags.ContainsJsx) ? "skip" : isJsxElement(n) || isJsxSelfClosingElement(n) || isJsxFragment(n) ? n : undefined

```

Comment:
The implicit import can be for "tslib" as well right so dont we need to handle/ignore that